README.md

```diff
@@ -19,6 +19,10 @@ npm run bootstrap
 
 `node-red-s1seven-api` only needs to be updated if the implementation of one of the endpoints changes, or if a dependency has a security vunerability.
 
+### Releasing the new packages
```

Comment:
That's a bit misleading.
This process is only needed for a first release on NPM of a new node-red node ?

Reply:
Nope, it needs to be done every time the package is released on NPM. Node-Red no longer automatically indexes packages, they need to be manually indexed using the process I wrote. I added this because I kept forgetting where I had to go to update them, so I thought the readme would be the best place. Details here: https://nodered.org/docs/creating-nodes/packaging

```
As of April 2020, the [the Node-RED Flow Library](https://flows.nodered.org/) is no longer able to automatically index and update nodes published on npm with the node-red keyword. Instead, a submission request has to be placed manually.
```
@getlarge

Reply:
i can't believe that!

Reply:
could you try the API which is used by the frontend you linked ?
here's the [code](https://github.com/node-red/flow-library/blob/master/routes/nodes.js#L292) and if you inspect the network request you can have an idea of what's expected by the express server.

Reply:
that might not work as they added csrf protection based on session, but that's worth a try!

Reply:
Sure, great idea!